tools/merge_note_registries.py

```
@@ -0,0 +1,137 @@
+        if not norm:
+            continue
+
+        if norm in url_registry or norm in additions:
+            continue
+
+        # Use the note's own date as the registry timestamp
+        date_val = post.get("date")
+        if date_val is None:
+            date_str = "unknown"
+        else:
+            date_str = str(date_val)
```

Ruff (0.16.1)

[warning] Use ternary operator `date_str = "unknown" if date_val is None else str(date_val)` instead of `if`-`else`-block

Replace `if`-`else`-block with `date_str = "unknown" if date_val is None else str(date_val)`

(SIM108)